### ISSUE TITLE
Unable to replicate results for Amazon's Style Transfer Acc. %

### ISSUE THREAD

**Issue by sharan21** [yixinL7/Direct-Style-Transfer#5]
I have trained a fasttext classifier on the same version of amazon data as recommended by this repo.
The accuracy is of this classifier quite low, around 78.5% on the test split. On using this classifier to generate the TST stats, I am getting only 45% accuracy on the outputs you have provided.

However, the paper reports 68% style transfer accuracy. I am not able to understand what the issue is, can you provide me with the hyperparameters for the fasttext model you used? Why is the accuracy so low?

It seems like the amazon dataset you have provided, on visual inspection, has quite a few mislabelled instances. Maybe that is the reason the pretrained classifier is struggling?
